src/index.js

```diff
@@ -14,6 +14,17 @@ const debug = (function () {
   }
 })()
 
+const createSwarm = (hub) => {
```

Comment:
What about: 
```js
const createSwarm = (hub) =>
  swarm(hub, {
    wrtc: !swarm.WEBRTC_SUPPORT ? require('wrtc') : null
    config: config.SWARM_CONFIG
  })
```

Reply:
Initializing the swOpts and filling each attribute seemed more explicit, what do you think ?

Reply:
nope I find it's more functional to declare and initialize once. It's less error prone

Reply:
By defining the object at the same time we declare it, it's also easier to know exactly what's inside the object